src/Visualization/Python/TransformVolumeData.py

```diff
@@ -218,6 +220,8 @@ def one_index_constraint(
 
     if arg.annotation in Mesh.values():
         return ElementArg("mesh")
+    elif arg.name == "element":
```

Comment:
For my understanding, what are you trying to do with this? Note that this returns the Python-representation of an element in a volume data file (`IterElements.Element`), not the C++ class `Element` as you might expect. If you want to expose another property of the `IterElements.Element` class, maybe just add that specific one here, as is already done with coords, jacobians, etc?

Either way, please add this to the docstring.

Reply:
Thanks for the info! I ended up not really needing this (yet). I'll leave it on a branch and we can figure out a good way to handle some filtering. The other possible use-case I had in mind is to get use the element->inertial maps, but then I realized that also needs the domain, etc. and just didn't/don't have time to figure that out right now. I'll drop it and bring it back once I actually have time to get it into a clean state :)